SPECS/uefistored.spec

```diff
@@ -47,13 +51,24 @@ ln -s uefistored %{buildroot}%{_sbindir}/varstored
 install -d %{buildroot}%{_datadir}/varstored/
 cp %{SOURCE1} %{buildroot}%{_datadir}/varstored/
 
+%check
+make -C tests fetch
```

Comment:
@stormi fetch goes out and grabs `munit`, the unit test harness.

I find it odd for rpmbuild to reach out to the network during `check`.  Do you have any suggestions on how this should actually be done?

Reply:
It's not just odd, it's completely forbidden :). Koji builds run offline, on purpose: build result must not depend on anything external.

There are two ways to provide this kind of dependency:
* The Right Way: package them as RPMs. If a RPM exists in CentOS 7 or EPEL repositories, I can easily add it to koji. I don't think there is one, though. Else we need to create the RPM ourselves and build it in koji.
* The Wrong Way But Sometimes Necessary: embed the tool in the `SOURCES/` directory. If the tool is written in an interpreted language and is simple enough (ie one single file), then this can be envisioned.

Reply:
Hmmm...  It's one C file and one C header.  MIT licensed.  Only compiled into the unit tests of course.  The RPM seems doable.  I'll just need to change the Makefile to allow the `munit` directory to be set.

Just out of curiosity, why don't we want any C source tarballs in `SOURCES/`?

Reply:
I thought there was an executable. Just two source C files (or a tarball containing both) is ok in SOURCES.